Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -216,7 +216,13 @@
     sprite: Objects/Weapons/Guns/Battery/laser_gun.rsi
   - type: StaticPrice
     price: 420
+  - type: Wieldable
+  - type: GunWieldBonus
+    minAngle: -20
+    maxAngle: -30
   - type: Gun
+    minAngle: 21
+    maxAngle: 32
```

Comment:
Can you explain why you used these angles and why it's copy-pasted 20 times please.

Reply:
Yeah sure. I made a bunch of guns wieldable when i added the assets, and theres gotta be some kind of upside to wielding. In order to avoid making any balance decisions myself about the accuracy of guns, as its not the purpose of the PR, I just referenced the gun component to find the default values of minangle and maxangle and made sure to set the qualities to compensate anywhere that I did not already see specified accuracies.

Reply:
Hopin I didnt accidentally skew any of the gun's accuracy or misunderstand how the fields work, but I checked it to make sure I didnt a couple times.